iXBRLViewerPlugin/viewer/src/js/inspector.js

```diff
@@ -97,18 +97,20 @@ Inspector.prototype.search = function (s) {
     $('tr', table).remove();
     viewer.clearRelatedHighlighting();
     $.each(results, function (i,r) {
-        viewer.highlightRelatedFact(r.fact);
-        var row = $('<tr>')
-            .click(function () { viewer.showAndSelectFact(r.fact) })
-            .mouseenter(function () { viewer.linkedHighlightFact(r.fact); })
-            .mouseleave(function () { viewer.clearLinkedHighlightFact(r.fact); })
-            .data('ivid', r.fact.id)
-            .appendTo($("tbody", table));
-        $('<td>')
-            .text(r.fact.getLabel("std")) // + " (" + r.score + ")" );
-            .appendTo(row);
+        if (i < 100) {
+            var row = $('<tr>')
+                .click(function () { viewer.showAndSelectFact(r.fact) })
+                .mouseenter(function () { viewer.linkedHighlightFact(r.fact); })
+                .mouseleave(function () { viewer.clearLinkedHighlightFact(r.fact); })
+                .data('ivid', r.fact.id)
+                .appendTo($("tbody", table));
+            $('<td>')
+                .text(r.fact.getLabel("std")) // + " (" + r.score + ")" );
+                .appendTo(row);
+        }
 
```

Comment:
Extra white space here

Reply:
Where exactly? 108 & 109 are indented as they're part of the statement started on 107.

Reply:
I think Derek means "Nit: extra newline on line 111 can be removed"